### DISCUSSION TITLE
Add Terraform repository for Langfuse self-hosting on Docs or README

### DISCUSSION THREAD

**Discussion by sotazum** [langfuse/langfuse#4664] [Ideas]
### Describe the feature or potential improvement

Hi Langfuse team,

First of all, Thank you for developing and operating such a wonderful product for LLMOps. I'd like to express my sincere gratitude.

Recently I created the repository of Terraform configuration of Langfuse self-hosting on Google Cloud to make the development experience improve for all Langfuse self-hosting users.

Could you consider to post this repo on your docs or README? 

URL: https://github.com/sotazum/langfuse-terraform

Look forward your reply. Thanks!

**Comment by marcklingen** [comment]
Merging this thread into the existing thread re terraform / deployment stack for GCP. @Steffen911 will have a look.

We'd aim to get this to use Langfuse v3 with the latest changes.

**Reply by marcklingen** [reply]
Existing thread: https://github.com/orgs/langfuse/discussions/4646
